.circleci/config.yml

```diff
@@ -30,6 +38,13 @@ jobs:
             - gradle-
       - run:
           command: ./gradlew check -x test
+      - persist_to_workspace:
+          root: .
+          paths:
+            - build
+            - src
+            - build.gradle.kts
```

Comment:
I wasn't sure which ones to add here - src and build.gradle.kts were added when testing - so I will remove